### PR TITLE
Add phenotype configuration notebook

### PR DESCRIPTION
**Pull request by roshankern**
In this PR we:
- add the configuration notebook for the phenotype process module
- make slight adjustments to the configuration notebook for the sbs process module

**Note:** Significant code for the phenotype process is migrated in this PR (ex cp_emulator) that is not directly used in this notebook but is left in this PR because it will eventually be needed when we create the snakemake for this module.

@mat10d This is ready for review!

### CODE REVIEW

**Review by mat10d — APPROVED** [review]
This looks good to merge. Some items to discuss, but we need to plan for it in the future rather than now, given that we are waiting on CP to make their code more accessible.

File: example_analysis/2.configure_sbs_process_params.ipynb [hunk 1]
```diff
@@ -40,15 +40,14 @@
   },
```
Comment (mat10d) [inline example_analysis/2.configure_sbs_process_params.ipynb:40]: <pre class="ql-syntax" spellcheck="false">Line #34.    from lib.shared.configuration_utils import random_cmap, CONFIG_FILE_HEADER
</pre><p>Was going to suggest you move this function out. Well done</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/cheeseman-lab/brieflow/pull/17/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>

Comment (mat10d) [inline example_analysis/2.configure_sbs_process_params.ipynb:40]: <pre class="ql-syntax" spellcheck="false">Line #5.    BASES = "GTAC"
</pre><p>Perfect</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/cheeseman-lab/brieflow/pull/17/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>

File: workflow/lib/external/cp_emulator.py
[the remark this answers is not in the document]
Comment (roshankern) [reply workflow/lib/external/cp_emulator.py:1]: As per our discussion, we will proceed with the cp_emulator approach for now, move to the [cp_measure](https://github.com/afermg/cp_measure) package short term, and hopefully integrate a direct Python version of Cell Profiler long term.

File: example_analysis/config/config.yml [hunk 2]
```diff
@@ -12,6 +12,18 @@
...
+  cell_diameter: 55.3
+  channel_names:
+  - dapi
+  - coxiv
+  - cenpa
+  - wga
+  cyto_channel: 1
+  cyto_model: cyto3
+  dapi_index: 0
+  foci_channel: 1
```
[the remark this answers is not in the document]
Comment (roshankern) [reply example_analysis/config/config.yml]: Done in https://github.com/cheeseman-lab/brieflow/pull/17/commits/f6bfc0bc990b2d4060fdc863a178a2759ce18223!

File: workflow/lib/shared/image_utils.py
[the remark this answers is not in the document]
Comment (roshankern) [reply workflow/lib/shared/image_utils.py:1]: Uh oh, forgot to talk about this during our meeting! Let's bring it up during our next convo around phenotype processing.

**Pull request merged**